Cargo.toml

```diff
@@ -60,9 +65,10 @@ rustc_version = "0.4.0"
 
 [features]
 binary = ["clap", "wild", "stderrlog"]
-default = ["binary", "filetime", "parallel", "zopfli"]
+default = ["binary", "filetime", "parallel", "zopfli", "sanity-checks"]
```

Comment:
Unfortunately this will enable the feature for both release and debug builds. It would be great if Cargo had a feature to enable dependencies only for debug mode, but sadly no such feature exists... what probably makes the most sense is to:
- Leave this feature out of the default set
- Update the CI to enable the `sanity-checks` feature when it runs tests

Reply:
I left it enabled to make sure that developers using debug builds generated with a simple `cargo run` or `cargo build` didn't accidentally opt-out of potentially useful assertions by default, but I agree that putting it out of the default feature set could also be a good idea. I don't have any hard preference towards either choice. Shall I change it? 😄

Reply:
The output validation has certainly been a great help for me when developing changes to the project, so it would be a shame to not have it on by default in debug builds. But I'm sure I can remember to manually enable it when I need.

As noted above though, there's no real harm in having it enabled in release since it shouldn't make any difference to the binary size. @AlexTMjugador what was your motivation for making this change? Does it build faster with the feature disabled?

Reply:
Yes, the first build is faster and takes less disk space because Cargo doesn't need to download and build the extra dependencies, even though it is smart enough to remove their traces from the generated executables.

Reply:
Right, makes sense.
So assuming people using it as a dependency won't be using the default options anyway, I would be in favour of keeping it enabled in default. But I won't mind if @shssoichiro prefers otherwise.

Reply:
I wouldn't say I have a strong preference, although I think I would lean toward keeping build times down when possible. As long as it's running in CI as a final "safety check", I'm content with that.

Reply:
Alright, I'm removing it from the default feature set and adding it to CI 👍